src/server/models/Challenge.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (sequelize, DataTypes) => {
+  const Challenge = sequelize.define('Challenge', {
+    name: DataTypes.STRING,
+    duration: DataTypes.INTEGER
+  });
+
+  Challenge.associate = (models) => {
```

Comment:
According to the drawn schema, I think the challenge model is meant to have moduleId.
The current implementation puts challengeId on the module model which might not be appropriate considering it is a **one-to-many** relationship

Reply:
Perhaps you can look into changing this to `belongsTo` so we can have appropriate accessor methods